docs/components/servo/_index.md

```diff
@@ -340,6 +340,13 @@ You can find additional assistance in the [Troubleshooting section](/appendix/tr
 
 {{< snippet "social.md" >}}
 
+## Related Services
+
+{{< cards >}}
+{{% card link="/services/data/" class="small" %}}
```

Comment:
Since a servo doesn’t output any data maybe this doesn’t make sense?

Reply:
Ah I see, many of these like the servo have the data capture option available on their config and I saw that you need to turn it on  in order to work with the data management service & the component. But it might not be the case that just because it has data capture methods available it will work with the service

Reply:
You're right that data capture does work with servos; I just mean that most people aren't using servos for their data since they don't actually put out any readings/data at all. Viam abstracts things such that you can get a stream of the positions a servo is *told* to go to, but not actually feedback from the hardware itself. So per offline discussion what I mean is just that, though people can technically use data capture with servos, it's not like an integral part of servo usage at all, so maybe we don't want to highlight it here.

Reply:
Thank you so much, this is really helpful!!